functions/_fzf_search_processes.fish

```diff
@@ -0,0 +1,21 @@
+function _fzf_search_processes --description "Search pid for all running commands"
+    set fzf_arguments --multi --ansi $fzf_processes_opts
+    set token (commandline --current-token)
+    # expand any variables or leading tilde (~) in the token
+    #set expanded_token (eval echo -- $token)
+
+    set pid_selected (
+        ps -A -opid,command | \
+        _fzf_wrapper --query $token  \
+                     --header-lines=1 \
+                     --preview='ps -o pid,user,%cpu,time,rss,command -p {1}' \
+                     $fzf_arguments | \
+        awk '{print $1}'
```

Comment:
how come you still need awk?

Reply:
Good point. The output from FZF is the entire line, but I'm trying to extract just the PID here to be useful for commands such as "kill". the `awk` here is simply extracting the pid from the selected line

Reply:
Ah so we want to reduce dependencies so you could just use standard fish functions to do this. Check out how I did it in `_fzf_search_git_log`